forest/cli/src/main.rs

```diff
@@ -5,20 +5,19 @@ mod cli;
 mod subcommand;
 
 use cli::{cli_error_and_die, Cli};
-
-use async_std::task;
 use forest_cli_shared::logger;
 use structopt::StructOpt;
 
-fn main() {
+#[tokio::main]
+async fn main() {
     // Capture Cli inputs
     let Cli { opts, cmd } = Cli::from_args();
 
     // Run forest as a daemon if no other subcommands are used. Otherwise, run the subcommand.
```

Comment:
This comment should be updated. `forest-cli` doesn't run the daemon.

Reply:
Fixed